phenopacketlab-ingest/src/main/java/org/monarchinitiative/phenopacketlab/ingest/cmd/IngestCommand.java

```diff
@@ -66,13 +88,21 @@ private static Optional<Path> createDataDirectory(Path destinationPath) {
         return Optional.of(destinationPath);
     }
 
-    private void downloadResources(Path dataDirectory) throws FileDownloadException {
+    private void downloadResources(Path dataDirectory, List<Path> toDelete) throws FileDownloadException, IOException {
+        String nciThesaurusZipName = "NciThesaurus.zip";
```

Comment:
Thanks, I commented out the code. The code for parsing NCIT flat file is likely a dead end; I realized too late that we can get the concepts from an OWL file in `PhenopacketLab/scripts/preprocess-ontologies.sh` script that is part of the ingest process.

Despite being a dead end, I want to keep the code around just in case, so I am not removing the unused classes altogether. We will reevaluate after more work has been done.